workers-docs/src/content/tutorials/build-a-serverless-function/_index.md

```diff
@@ -181,7 +181,7 @@ async function handleRequest(request) {
 
 The `landing` variable, which is a static HTML string, sets up an `input` tag and a corresponding `button`, which calls the `generate` function. This function will make an HTTP `POST` request back to your serverless function, allowing you to see the corresponding QR code image data inside of your browser's network inspector:
 
-<img src="./media/demo.png" alt="demo icon"/>
+![](/tutorials/build-a-serverless-function/media/demo.png)
```

Comment:
Is it worth keeping the alt text?

Reply:
Images like this must contain alt text. However, in this case, the alt text is incorrect.

![Screenshot of QR code generator working in the browser](https://user-images.githubusercontent.com/154613/82259892-48f65200-992a-11ea-974c-23307363f10e.png)

Based on what I’m seeing here, I’d suggest something more like “QR code generator working in the browser”.

Reply:
I added this great suggestion thanks :)